packages/config/src/library/v2/common.ts

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface MudConfigInput {
+  /** This interface can be extended by MUD config plugins */
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface, @typescript-eslint/no-unused-vars
+export interface MudResolvedConfigOutput<configInput extends MudConfigInput = MudConfigInput> {
+  /** This interface can be extended by MUD config plugins */
+}
+
+export interface MudConfigOutput<configInput extends MudConfigInput = MudConfigInput> {
+  resolved: MudResolvedConfigOutput<configInput>;
+}
```

Comment:
do we still use these interfaces? should they be `type` instead of `interface`?

Reply:
nope, removing this